sources/packages/api/src/route-controllers/application/application.controller.ts

```diff
@@ -93,11 +94,28 @@ export class ApplicationController extends BaseController {
       studentFiles,
     );
 
-    await this.workflow.startApplicationAssessment(
+    const assessmentWorflow = await this.workflow.startApplicationAssessment(
       submissionResult.data.data.workflowName,
       createdApplication.id,
     );
 
+    // TODO: Once we have the application status we should run the create/associate
+    // under a DB transation to ensure that, if the workflow fails to start we would
+    // be rolling back the transaction and returning an error to the student.
+    const workflowAssociationResult =
+      await this.applicationService.associateAssessmentWorkflow(
+        createdApplication.id,
+        assessmentWorflow.id,
+      );
+
+    if (workflowAssociationResult.affected !== 1) {
```

Comment:
can we add enum for these statuses, or add a comment, like to quickly understand, what does status 1 and 0 means

Reply:
I believe that an enum does not make sense because 1 just represents the expected number of affected rows. I will add a comment.

Reply:
Comment added.